### PR TITLE
fix: HTTP links to HTTPS

### PR DESCRIPTION
**Pull request by nschonni**
Changed some localhost links to code fences too

### CODE REVIEW

File: files/en-us/glossary/imap/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ IMAP (Internet Message Access Protocol) is a {{Glossary("protocol")}} used to re
 
 Unlike POP3, IMAP allows multiple simultaneous connections to one mailbox. Clients accessing a mailbox can receive information about state changes made from other clients. IMAP also provides a mode for clients to stay connected and receive information on demand.
 
-Mark Crispin initially developed IMAP in 1986 as _Interim Mail Access Protocol_. IMAP4 revision 1 is the current version, defined by [RFC 3501](http://www.faqs.org/rfcs/rfc3501.html).
+Mark Crispin initially developed IMAP in 1986 as _Interim Mail Access Protocol_. IMAP4 revision 1 is the current version, defined by [RFC 3501](https://www.ietf.org/rfc/rfc3501.html).
```
Comment (ddbeck) [inline files/en-us/glossary/imap/index.md:14]: In mdn/browser-compat-data, we've taken to using the rfc-editor.org URLs as (more or less) canonical.

```suggestion
Mark Crispin initially developed IMAP in 1986 as _Interim Mail Access Protocol_. IMAP4 revision 1 is the current version, defined by [RFC 3501](https://www.rfc-editor.org/info/rfc3501).
```

File: files/en-us/learn/server-side/django/deployment/index.md [hunk 2]
```diff
@@ -251,36 +251,36 @@ The final steps are to copy your application into this local project directory
...
-            new file:   catalog/__init__.py
+            modified:   .gitignore
+            new file:   catalog/__init__.py
             ...
-            new file:   catalog/migrations/0001_initial.py
+            new file:   catalog/migrations/0001_initial.py
             ...
-            new file:   templates/registration/password_reset_form.html
+            new file:   templates/registration/password_reset_form.html
     ```
```
Comment (ddbeck) [inline files/en-us/learn/server-side/django/deployment/index.md:269]: It's not clear what's changing here precisely. Is this still the actual output from Git?

**Review by ddbeck — CHANGES_REQUESTED** [review]
Thanks for this PR, @nschonni.

I've reviewed a bunch of these, with a couple of comments in line, but not the whole PR. Would it be difficult for you to update the PR to separate out the white space changes to its own commit or separate PR? I'm not opposed the white space changes, but as it stands, it's a bit of a challenge to spot the URL changes in between.

I don't want to stand in the way of good work, but if this could be made easier to review, that would be ideal.

File: files/en-us/glossary/imap/index.md (hunk 1 above)
Comment (nschonni) [reply files/en-us/glossary/imap/index.md]: There are other hits for the IETF site in the repo, but i'll leave you to do that other cleanup later

File: files/en-us/learn/server-side/django/deployment/index.md (hunk 2 above)
Comment (nschonni) [reply files/en-us/learn/server-side/django/deployment/index.md:269]: These are non-space characters changing to spaces

**Comment by nschonni** [comment]
> I've reviewed a bunch of these, with a couple of comments in line, but not the whole PR. Would it be difficult for you to update the PR to separate out the white space changes to its own commit or separate PR? I'm not opposed the white space changes, but as it stands, it's a bit of a challenge to spot the URL changes in between.

No, they can't be separated out, that out entail redoing the entire PR. If it bothers you though, you can close. I've been cleaning them up as I touch the documents as the issue is everywere going back to the wiki markup migration

**Comment by ddbeck** [comment]
> No, they can't be separated out, that out entail redoing the entire PR. If it bothers you though, you can close. I've been cleaning them up as I touch the documents as the issue is everywere going back to the wiki markup migration

OK, that's fine. I'll review it all together. For future reference, breaking commits into specific tasks (e.g., white space as one commit, substantive changes as another) helps speed up reviews. We typically squash merges here, as well, so it'll get collapsed at the end.

File: files/en-us/learn/server-side/django/forms/index.md [hunk 3]
```diff
@@ -641,17 +641,17 @@ The author create, update, and delete pages are now ready to test (we won't both
...
-First, log in to the site with an account that has whatever permissions you decided are needed to access the author editing pages.
+First, log in to the site with an account that has whatever permissions you decided are needed to access the author editing pages.
 
-Then navigate to the author create page, _<http://127.0.0.1:8000/catalog/author/create/>_, which should look like the screenshot below.
+Then navigate to the author create page, `http://127.0.0.1:8000/catalog/author/create/`, which should look like the screenshot below.
 
 ![Form Example: Create Author](forms_example_create_author.png)
 
-Enter values for the fields and then press **Submit** to save the author record. You should now be taken to a detail view for your new author, with a URL of something like _http\://127.0.0.1:8000/catalog/author/10_.
+Enter values for the fields and then press **Submit** to save the author record. You should now be taken to a detail view for your new author, with a URL of something like _http\://127.0.0.1:8000/catalog/author/10_.
```
Comment (ddbeck) [inline files/en-us/learn/server-side/django/forms/index.md:650]: For consistency with above.

```suggestion
Enter values for the fields and then press **Submit** to save the author record. You should now be taken to a detail view for your new author, with a URL of something like `http://127.0.0.1:8000/catalog/author/10`.
```

**Comment by nschonni** [comment]
I hear you, and totally believe in one singe type of change or a bunch of changes for a single file in one commit. With cleaning up the non-spaces, it only becomes feasable to include in other changes, since each change needs to be manually reviewed to see if the NBSPs are intentional or not, which only happens when the file is opened for another change. I really with the "Hide Whitespace" option on the Files tab would also ignore these changes, but it also seems to treat them as more than normal whitespace

File: files/en-us/web/xslt/transforming_xml_with_xslt/for_further_reading/index.md [hunk 4]
```diff
@@ -103,7 +103,7 @@ tags:
 
 - **XSL-List**
 
-  - **Subscribe**: <http://www.mulberrytech.com/xsl/xsl-list/>
+  - **Subscribe**: <https://www.mulberrytech.com/xsl/xsl-list/>
```
Comment (ddbeck) [inline files/en-us/web/xslt/transforming_xml_with_xslt/for_further_reading/index.md:106]: This redirects here through an interstitial page.

```suggestion
  - **Subscribe**: <https://www.mulberrytech.com/xsl/xsl-list/index.html>
```

**Review by ddbeck — APPROVED** [review]
Thank you, @nschonni. Just a couple little suggestions for you and we'll be good to go.

**Pull request merged**